player/lua/ytdl_hook.lua

```diff
@@ -43,10 +43,29 @@ end
 
 -- youtube-dl JSON name to mpv tag name
 local tag_list = {
+    ["artist"]          = "artist",
+    ["album"]           = "album",
+    ["album_artist"]    = "album_artist",
```

Comment:
Shouldn't this match capitalization of tags in options? See: 
https://github.com/mpv-player/mpv/blob/f6d931301bd226239d95f59e3f18abfeab692ee9/options/options.c#L1103-L1109

Reply:
Implementation detail of `mp_tags_set_bstr`, they're case insensitive.